src/main.test.ts

```diff
@@ -183,7 +183,7 @@ describe('main', () => {
 	});
 
 	test('reports interaction errors', async () => {
-		const interactionError = new Error('Failed to handle ineracion. This is a test.');
+		const interactionError = new Error('Failed to handle ineraction. This is a test.');
```

Comment:
```suggestion
		const interactionError = new Error('Failed to handle interaction. This is a test.');
```
Almost fixed the spelling error, just one more 't'.